unime/package.json

```diff
@@ -32,6 +32,7 @@
     "@iconify-json/emojione": "^1.1.7",
     "@iconify-json/heroicons": "^1.1.11",
     "@iconify-json/lucide": "^1.1.112",
+    "@iconify-json/mdi": "^1.1.64",
```

Comment:
I don't see this used anywhere (searching for ~icons/mdi).

Reply:
I believe it was used for an icon, but not anymore. Will remove it 👍